CIScripts/Test/Tests/VTestScenarios.Tests.ps1

```diff
@@ -1,13 +1,15 @@
 Param (
-    [Parameter(Mandatory=$true)] [string] $TestbedAddr
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr,
+    [Parameter(Mandatory=$true)] [string] $TestConfigurationFile   
 )
 
 . $PSScriptRoot\..\Utils\CommonTestCode.ps1
+. $PSScriptRoot\..\Utils\ComponentsInstallation.ps1
 . $PSScriptRoot\..\TestConfigurationUtils.ps1
 . $PSScriptRoot\..\..\Common\VMUtils.ps1
 . $PSScriptRoot\..\PesterHelpers\PesterHelpers.ps1
 
-. $PSScriptRoot\..\GetTestConfigurationJuni.ps1
+. $PSScriptRoot\..\($TestConfigurationFile)
```

Comment:
$TestConfigurationFile is passed as a full path, so just `. $TestConfigurationFile` should work.